skiko/src/jvmMain/kotlin/org/jetbrains/skiko/redrawer/LinuxOpenGLRedrawer.kt

```diff
@@ -95,6 +125,12 @@ internal class LinuxOpenGLRedrawer(
             .filterNot(LinuxOpenGLRedrawer::isDisposed)
             .filter { it.layer.isShowing }
 
+        private val drawDispatcher = Executors.newSingleThreadExecutor {
```

Comment:
So all drawing happens in this thread, no matter how many open windows we got?

Reply:
Maybe write comment, and also on how we ensure that no races on drawing context happen?